src/main/java/org/jenkinsci/test/acceptance/plugins/git/GitRepo.java

```diff
@@ -57,6 +57,7 @@
 
     public GitRepo() {
         dir = initDir();
+        git("config", "--global", "init.defaultBranch", "master");
```

Comment:
This does not look safe: if a developer runs this on their host OS, it will overwrite `~/.gitconfig`!